librz/bp/bp_plugin.c

```diff
@@ -45,24 +47,32 @@ RZ_API bool rz_bp_plugin_del(RzBreakpoint *bp, RZ_BORROW RZ_NONNULL RzBreakpoint
  */
 RZ_API int rz_bp_use(RZ_NONNULL RzBreakpoint *bp, RZ_NONNULL const char *name) {
 	rz_return_val_if_fail(bp && name, false);
-	RzListIter *iter;
-	RzBreakpointPlugin *h;
-	rz_list_foreach (bp->plugins, iter, h) {
+	RzIterator *iter = ht_sp_as_iter(bp->plugins);
+	RzBreakpointPlugin **val;
+	rz_iterator_foreach(iter, val) {
+		RzBreakpointPlugin *h = *val;
 		if (!strcmp(h->name, name)) {
 			bp->cur = h;
+			rz_iterator_free(iter);
 			return true;
 		}
 	}
+	rz_iterator_free(iter);
 	return false;
 }
 
 // TODO: deprecate
 RZ_API void rz_bp_plugin_list(RzBreakpoint *bp) {
-	RzListIter *iter;
+	RzIterator *iter = ht_sp_as_iter(bp->plugins);
+	RzList *plugin_list = rz_list_new_from_iterator(iter);
```

Comment:
Check for NULL

Reply:
Maybe rename the function into `_print()` while you are on it? For consistency with other plugins, also remove the comment about deprecation?